packages/test-runner-core/src/test-session/TestSessionStatus.ts

```diff
@@ -0,0 +1,13 @@
+export type TestSessionStatus = 'SCHEDULED' | 'INITIALIZING' | 'STARTED' | 'FINISHED';
```

Comment:
Why not a const enum?

Reply:
I avoid enums because they cause trouble when using tools that use single file transforms, and when it's part of the public API I want people to be able to use it just as a string (ex. in a config).